certified-connectors/HubPlanner/readme.md

```diff
@@ -0,0 +1,25 @@
+# Hub Planner
+Hub Planner is a resource management SaaS platform helping global teams schedule and manage their resources across projects.
```

Comment:
Kindly update the readme.md file as per the template in the below path. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md